tools/travis-ci-script.sh

```diff
@@ -24,7 +24,7 @@ git checkout HEAD@{1} src/message
 git diff --staged
 git commit -m "update autogenerated files"
 
-if [ ! -z ${TRAVIS_PULL_REQUEST} ] && [ ${TRAVIS_PULL_REQUEST} == "false" ]; then
```

Comment:
can we leave the ! -z (string not empty) conditional? I believe this was added to fail gracefully when running this script locally.